.golangci.yml

```diff
@@ -17,3 +17,4 @@ linters:
   enable-all: true
   disable:
     - maligned
+    - lll
```

Comment:
Would prefer that we relax this constraint rather than disabling it altogether.
i.e; 
```yaml
 lll:
# max line length, lines longer will be reported. Default is 120. '\t' is counted as 1 character.
line-length: 200
```

should pass and still we guard against unwanted code

Reply:
ok. We discussed it and I rest my case. Let's disable it then